ape_safe/accounts.py

```diff
@@ -157,6 +182,16 @@ def safe_tx_def(self) -> Type[SafeTx]:
         )
 
     def create_safe_tx(self, txn: Optional[TransactionAPI] = None, **safe_tx_kwargs) -> SafeTx:
+        """
+        Create the Safe transaction.
+
+        Args:
+            txn (Optional[``TransactionAPI``]): The transaction
+            **safe_tx_kwargs: The safe transactions specifications, such as ``submitter``.
+
+        Returns:
+            ``:class:~ape_safe.client.SafeTx``: The Safe Transaction to be used.
+        """
         safe_tx = {}
```

Comment:
nit: This whole deal could be a dict literal:

```python
        safe_tx = {
            "to": safe_tx_kwargs.get(
                "to", txn.receiver if txn else self.address  # Self-call, e.g. rejection
            ),
            "value": safe_tx_kwargs.get("value", txn.value if txn else 0),
            "data": safe_tx_kwargs.get("data", txn.data if txn else b""),
            "nonce": safe_tx_kwargs.get("nonce", self.next_nonce),
            "operation": safe_tx_kwargs.get("operation", 0),
            "safeTxGas": safe_tx_kwargs.get("safeTxGas", 0),
            "baseGas": safe_tx_kwargs.get("baseGas", 0),
            "gasPrice": safe_tx_kwargs.get("gasPrice", 0),
            "gasToken": safe_tx_kwargs.get(
                "gasToken", "0x0000000000000000000000000000000000000000"
            ),
            "refundReceiver": safe_tx_kwargs.get(
                "refundReceiver", "0x0000000000000000000000000000000000000000"
            ),
        }
```

Reply:
also - you can use ZERO_ADDRESS for the zero address values

Reply:
Wonder if you could do:

```py
        safe_tx = {
            "to", txn.receiver if txn else self.address,  # Self-call, e.g. rejection
            "value": txn.value if txn else 0,
            "data": txn.data if txn else b"",
            "nonce": self.next_nonce,
            "operation": 0,
            ...,
            **safe_tx_kwargs,  # Overrides above
        }